driver-sync/src/test/functional/com/mongodb/client/unified/EventMatcher.java

```diff
@@ -313,10 +313,12 @@ private static boolean serverDescriptionChangedEventMatches(final BsonDocument e
             return true;
         }
         String newType = expectedEventContents.getDocument("newDescription").getString("type").getValue();
-        //noinspection SwitchStatementWithTooFewBranches
         switch (newType) {
             case "Unknown":
                 return event.getNewDescription().getType() == ServerType.UNKNOWN;
+            case "LoadBalancer": {
+                return event.getNewDescription().getType() == ServerType.LOAD_BALANCER;
+            }
```

Comment:
This change is supposed to fix https://spruce.mongodb.com/task/mongo_java_driver_tests_serverless__os~ubuntu_jdk~jdk17_serverless_test_d375cd45b2cb60cf4400e8f0ed250ef933af9604_23_11_01_16_11_45/tests?execution=0&sortBy=STATUS&sortDir=ASC.

Reply:
And does it?  :)

Reply:
No, not initially. Hopefully, together with the new logging code it will.

Reply:
Ok, now it worked.